[2024-09-09T20:14:18.485+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_daily_order_update_dag.run_fetch_raw_amazon_returns_report scheduled__2024-09-09T18:00:00+00:00 [queued]>
[2024-09-09T20:14:18.502+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_daily_order_update_dag.run_fetch_raw_amazon_returns_report scheduled__2024-09-09T18:00:00+00:00 [queued]>
[2024-09-09T20:14:18.503+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-09-09T20:14:18.541+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): run_fetch_raw_amazon_returns_report> on 2024-09-09 18:00:00+00:00
[2024-09-09T20:14:18.549+0000] {standard_task_runner.py:57} INFO - Started process 183 to run task
[2024-09-09T20:14:18.555+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'run_daily_order_update_dag', 'run_fetch_raw_amazon_returns_report', 'scheduled__2024-09-09T18:00:00+00:00', '--job-id', '245', '--raw', '--subdir', 'DAGS_FOLDER/run_daily_order_update.py', '--cfg-path', '/tmp/tmpel0eoo_s']
[2024-09-09T20:14:18.584+0000] {standard_task_runner.py:85} INFO - Job 245: Subtask run_fetch_raw_amazon_returns_report
[2024-09-09T20:14:18.705+0000] {task_command.py:415} INFO - Running <TaskInstance: run_daily_order_update_dag.run_fetch_raw_amazon_returns_report scheduled__2024-09-09T18:00:00+00:00 [running]> on host 2c451b520589
[2024-09-09T20:14:18.871+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_daily_order_update_dag' AIRFLOW_CTX_TASK_ID='run_fetch_raw_amazon_returns_report' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-09T18:00:00+00:00'
[2024-09-09T20:14:18.875+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-09T20:14:18.877+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker exec python-scripts-container python3 /opt/python_scripts/run_fetch_raw_amazon_returns_report.py']
[2024-09-09T20:14:18.893+0000] {subprocess.py:86} INFO - Output:
[2024-09-09T20:14:20.094+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-09-09T20:14:20.098+0000] {subprocess.py:93} INFO -   File "/opt/python_scripts/run_fetch_raw_amazon_returns_report.py", line 15, in <module>
[2024-09-09T20:14:20.099+0000] {subprocess.py:93} INFO -     start_date = values[0][0] + timedelta(seconds=1)
[2024-09-09T20:14:20.099+0000] {subprocess.py:93} INFO - TypeError: can only concatenate str (not "datetime.timedelta") to str
[2024-09-09T20:14:20.184+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-09-09T20:14:20.239+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-09-09T20:14:20.250+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=run_daily_order_update_dag, task_id=run_fetch_raw_amazon_returns_report, execution_date=20240909T180000, start_date=20240909T201418, end_date=20240909T201420
[2024-09-09T20:14:20.321+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 245 for task run_fetch_raw_amazon_returns_report (Bash command failed. The command returned a non-zero exit code 1.; 183)
[2024-09-09T20:14:20.349+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-09T20:14:20.381+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-09T20:27:45.245+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: run_daily_order_update_dag.run_fetch_raw_amazon_returns_report scheduled__2024-09-09T18:00:00+00:00 [queued]>
[2024-09-09T20:27:45.249+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: run_daily_order_update_dag.run_fetch_raw_amazon_returns_report scheduled__2024-09-09T18:00:00+00:00 [queued]>
[2024-09-09T20:27:45.250+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-09-09T20:27:45.259+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): run_fetch_raw_amazon_returns_report> on 2024-09-09 18:00:00+00:00
[2024-09-09T20:27:45.287+0000] {standard_task_runner.py:57} INFO - Started process 317 to run task
[2024-09-09T20:27:45.295+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'run_daily_order_update_dag', 'run_fetch_raw_amazon_returns_report', 'scheduled__2024-09-09T18:00:00+00:00', '--job-id', '252', '--raw', '--subdir', 'DAGS_FOLDER/run_daily_order_update.py', '--cfg-path', '/tmp/tmpja_id9i7']
[2024-09-09T20:27:45.299+0000] {standard_task_runner.py:85} INFO - Job 252: Subtask run_fetch_raw_amazon_returns_report
[2024-09-09T20:27:45.377+0000] {task_command.py:415} INFO - Running <TaskInstance: run_daily_order_update_dag.run_fetch_raw_amazon_returns_report scheduled__2024-09-09T18:00:00+00:00 [running]> on host 2c451b520589
[2024-09-09T20:27:45.461+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='run_daily_order_update_dag' AIRFLOW_CTX_TASK_ID='run_fetch_raw_amazon_returns_report' AIRFLOW_CTX_EXECUTION_DATE='2024-09-09T18:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-09T18:00:00+00:00'
[2024-09-09T20:27:45.463+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-09T20:27:45.464+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'docker exec python-scripts-container python3 /opt/python_scripts/run_fetch_raw_amazon_returns_report.py']
[2024-09-09T20:27:45.470+0000] {subprocess.py:86} INFO - Output:
[2024-09-09T20:27:46.713+0000] {subprocess.py:93} INFO - Traceback (most recent call last):
[2024-09-09T20:27:46.716+0000] {subprocess.py:93} INFO -   File "/opt/python_scripts/run_fetch_raw_amazon_returns_report.py", line 15, in <module>
[2024-09-09T20:27:46.716+0000] {subprocess.py:93} INFO -     start_date = values[0][0] + timedelta(seconds=1)
[2024-09-09T20:27:46.717+0000] {subprocess.py:93} INFO - TypeError: can only concatenate str (not "datetime.timedelta") to str
[2024-09-09T20:27:46.895+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2024-09-09T20:27:46.950+0000] {taskinstance.py:1943} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2024-09-09T20:27:46.963+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=run_daily_order_update_dag, task_id=run_fetch_raw_amazon_returns_report, execution_date=20240909T180000, start_date=20240909T202745, end_date=20240909T202746
[2024-09-09T20:27:47.058+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 252 for task run_fetch_raw_amazon_returns_report (Bash command failed. The command returned a non-zero exit code 1.; 317)
[2024-09-09T20:27:47.099+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-09-09T20:27:47.174+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
